client/app/account/user/security/users/add/user-users-add.html

```diff
@@ -87,6 +87,30 @@
                     </span>
                 </div>
 
+                <div class="form-group"
+                     data-ng-class="{ 'has-error': $ctrl.userAccountUsersAddForm.group.$touched && $ctrl.userAccountUsersAddForm.group.$invalid }">
+
+                    <label class="control-label"
+                           for="group"
+                           data-translate="user_users_role">
+                    </label>
+                    <select
+                           class="form-control"
+                           id="group"
+                           name="group"
+                           required
+                           data-ng-model="$ctrl.user.group">
+                                <option value="ADMIN" data-translate="user_roles_admin"/>
+                                <option value="DEFAULT" data-translate="user_roles_regular"/>
+                                <option value="UNPRIVILEGED" data-translate="user_roles_unprivileged"/>
+                    </select>
+                    <span class="help-block"
```

Comment:
You should display a message if `$error.required` exist: `<span data-ng-message="required" data-translate="YOUR_OWN_KEY"></span>`.